tests/unit/h/accounts/schemas_test.py

```diff
@@ -72,7 +74,6 @@ def test_it_is_valid_when_authorized_users_email(
         schemas.unique_email(dummy_node, "elliot@bar.com")
 
 
-@pytest.mark.usefixtures("user_model")
```

Comment:
Bad tests: mocking `models.User`. We shouldn't mock models. In this instance the presence of this mock means that once I added an SQL query of the `User` model to the schema code all the tests started failing because SQLAlchemy crashes if you pass a mock object to a query instead of a real model object.

The reason the tests are mocking the model is that the model has methods on it like `get_by_username()`, `get_by_email()`, etc. Models shouldn't have methods like this, those belong on services.